plugins/vlc-video/vlc-video-source.c

```diff
@@ -489,7 +496,7 @@ static void vlcs_update(void *data, obs_data_t *settings)
 	obs_data_array_t *array;
 	const char *behavior;
 	size_t count;
-	int network_caching;
+	int network_caching, track_index;
```

Comment:
Style:  Avoid declaring multiple variables on a single line.

```suggestion
	int network_caching;
	int track_index;
```

Reply:
done